src/main/webapp/consentGroupDocuments/ConsentGroupDocuments.js

```diff
@@ -155,6 +154,8 @@ export const ConsentGroupDocuments = hh(class ConsentGroupDocuments extends Comp
         actionLabel: 'Yes'
       }, []),
       Documents({
+        showSpinner: this.props.showSpinner,
+        hideSpinner: this.props.hideSpinner,
```

Comment:
Similar question here where we pass the parent component's spinner to the child.

Reply:
That description seems like *this* story to me. The passing in of the parent spinner is new code here. It would be just as easy to wrap `Documents` than it is to introduce an anti-pattern.